package.json

```diff
@@ -105,5 +104,8 @@
   "dependencies": {
     "heap": "^0.2.6",
     "lodash": "^4.17.21"
+  },
+  "directories": {
+    "test": "test"
```

Comment:
Used to specify where all test code is present. (Automatically added during the release process by npm).